lib/game.rb

```diff
@@ -1,22 +1,32 @@
 class Game
   attr_reader :current_player
 
-  def initialize(player1, player2)
+  def initialize(board, display, player1, player2)
+    @board = board
+    @display = display
     @player1 = player1
     @player2 = player2
     @current_player = player1
   end
 
-  def prompt_make_move
+  def prompt_for_move
```

Comment:
Should this method be private?

Reply:
Taking into considering the feedback below, I believe this method is redundant. However, I will consider making other methods private. Thank you.